recipes-core/init-ifupdown/init-ifupdown_1.0.bbappend

```diff
@@ -0,0 +1,11 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
+
+# interfaces.wait_iface file is based on commit 733ae41f20983a7e6f1c147188aa6b4db951d05b.
+SRC_URI += "file://interfaces.wait_iface \
```

Comment:
Could you name the file `interface`, and put in `recipes-core/init-ifupdown/init-ifupdown-1.0/raspberrypi3-64/interface` ? Yocto will consider this.
Then, you don't need to replace the `interface` file at `do_install_append_raspberrypi3-64()`.

Reply:
Like this. http://git.yoctoproject.org/cgit/cgit.cgi/poky/tree/meta/recipes-core/init-ifupdown/init-ifupdown-1.0

Reply:
@hiraku-wfs thanks. I've updated directory structure. Also removed eth1, atml0,bnep0, and usb0  from interfaces file which don't exist raspberry pi 3 by default.

Reply:
Thanks!